### ISSUE TITLE
datepicker when locale ar in update or create set nil value 

### ISSUE THREAD

**Issue by alibaobaid** [sferik/rails_admin#3327]
I get deep into a lot of solutions all of them solve that by format but in my case formate is not solving the issue finally I figure out that my issue caused because Datepicker passed numbers in `٠١٢٣٤٥٦٧٨٩` style and rails can not convert that style to date.
can you help me by fixing this issue to force Datepicker  to pass the date in 0123456789 numbers type even when the locale ar
thanks in advance

**Comment by amed** [comment]
Check: https://github.com/sferik/rails_admin/issues/2535#issuecomment-760541137


Sorry for late reply

**Comment by riulsudario** [comment]
You have to add:
  ```
  field :field_datetime do
    strftime_format do
      '%d-%m-%Y %H:%M:%S'
    end
  end
```
in your `rails_admin` module, remember to do this for each `DateTime` field. It is working on Ruby 2.7.0 and Rails 6.0.3.

This example is for `DateTime`, but it also works for `Date` fields, however, it would look like this:
  ```
  field :field_date do
    strftime_format do
      '%d-%m-%Y'
    end
  end
```

**Comment by mshibuya** [comment]
https://github.com/sferik/rails_admin/commit/01e8d5fc8ec94e68af6fdbd80759a751cd83f74a should have addressed this.

**Comment by alibaobaid** [comment]
@mshibuya the issue still exists after I update the gem to 2.2.1 
I try to store a date column when the locale ar I got null stored nested
![Screenshot from 2021-10-03 16-38-25](https://user-images.githubusercontent.com/40620739/135756117-f905ef93-b128-453f-8e66-7a691c63be15.png)
![Screenshot from 2021-10-03 16-37-47](https://user-images.githubusercontent.com/40620739/135756125-6ea9e186-758d-425a-b97b-76d840e2328a.png)

**Comment by mshibuya** [comment]
It's not released yet, you need to use the master branch.

**Comment by alibaobaid** [comment]
@mshibuya this issue got fixed in alpha but now showed again in 3.0.0. beta,  please check it

**Comment by mshibuya** [comment]
That's unlikely, what I released as 3.0.0.beta of course includes this change. Could you explain more about the issue you're having?

**Comment by alibaobaid** [comment]
@mshibuya when I select a date in date picker it passes nil when the local ar
